### DISCUSSION TITLE
Paddle lite model was not able to recognise the texts.

### DISCUSSION THREAD

**Discussion by vasujoshi111** [PaddlePaddle/PaddleOCR#12332] [Q&A]
![Recog_not](https://github.com/PaddlePaddle/PaddleOCR/assets/71640167/d40a2b31-b44d-445d-96ab-93db68b14027)
In this image was taken after the lite detection model. Recognised text was only 1. But if we apply this to .pdparams in windows giving correct results. 
We have not done any quantization on models and used exact models of pdmodels for inferencing. Please help me in debugging this.

**Comment by vasujoshi111** [comment]
I checked like argmax value after recognition.predidt has run, and got that only for 1 has recognised and given 93 value for that. For rest # was coming.

**Comment by cuicheng01** [comment]
What is your prediction command?

**Comment by vasujoshi111** [comment]
I am using paddle Android demo app. Like in pipeline.cc(clspredictor->predict)
